modules/claims_api/lib/claims_api/v1/poa_pdf_constructor/organization.rb

```diff
@@ -45,7 +45,7 @@ def page2_options(data)
             "#{base_form}.SocialSecurityNumber_FirstThreeNumbers[0]": data.dig('veteran', 'ssn')[0..2],
             "#{base_form}.SocialSecurityNumber_SecondTwoNumbers[0]": data.dig('veteran', 'ssn')[3..4],
             "#{base_form}.SocialSecurityNumber_LastFourNumbers[0]": data.dig('veteran', 'ssn')[5..8],
-            "#{base_form}.I_Authorize[1]": data['recordConcent'] == true ? 1 : 0,
+            "#{base_form}.I_Authorize[1]": data['recordConsent'] == true ? 1 : 0,
```

Comment:
@mchristiansonVA  This value `recordConcent` appears to still be in the `poa_form_builder_spec.rb` file at line 20.  Does that also need to get updated? (Looks like the testa are still passes so not sure about that)

Reply:
Good catch! Updated for consistency, though as you mentioned it doesn't seem to matter for the tests.